yappi/_yappi.c

```diff
@@ -1283,6 +1390,17 @@ profile_event(PyObject *self, PyObject *args)
     Py_RETURN_NONE;
 }
 
+static int
+profile_event_c(PyObject *self, PyFrameObject *frame, int what,

```

Comment:
I still do not think we need this function. Also we modify `c_profilefunc` to be this function but there are explicit checks like this in the code:

```
if (ts->c_profilefunc != _yapp_callback)
```
This might cause problems?